.circleci/config.yml

```diff
@@ -2,26 +2,26 @@ version: 2
 jobs:
   build_push:
     docker:
-    - image: cimg/base:stable-20.04
+      - image: cimg/base:current
```

Comment:
`current` is the replacement of `stable`, see https://circleci.com/developer/images/image/cimg/base#image-tags

Reply:
circleci naming :joy:

Reply:
😬😬